test/UnitTest/Microsoft.AspNet.OData.Test.Shared/Formatter/InheritanceTests.cs

```diff
@@ -192,6 +192,225 @@ public async Task Can_Patch_Entity_In_Inheritance()
             Assert.False((bool)result.CanDoAWheelie);
         }
 
+        [Fact]
+        public async Task Can_Patch_Entity_In_Inheritance_DerivedEngine()
```

Comment:
Please validate that the type, as well as properties, are updated in each of these tests